package.json

```diff
@@ -26,69 +26,44 @@
   "homepage": "https://github.com/terrestris/geostyler-openlayers-parser#readme",
   "dependencies": {
     "@terrestris/ol-util": "2.2.0",
-    "geostyler-style": "0.14.3",
+    "geostyler-style": "1.1.0",
     "lodash": "4.17.11"
   },
   "peerDependencies": {
     "ol": "~5.0"
   },
   "scripts": {
-    "build": "tsc -p tsconfig.build.json && npm run build:browser",
+    "build": "tsc -p tsconfig.json && npm run build:browser",
     "build:browser": "webpack --config browser-build.config.js",
-    "start:dev": "webpack --config dev-build.config.js --mode development --watch",
```

Comment:
Is there a reason you removed the dev-build?

Reply:
Just to keep it in line with the other projects and to reduce the amount of config files.